### PR TITLE
Making better preview of Skill and Bot Card

### LINKED ISSUE #3403: Unnecessary text on create skill card
**Actual Behaviour**
The text on create card is barely visible and not looking aesthetic.

**Expected Behaviour**

Text should be removed as icon '+' is self-explanatory.

**Steps to reproduce it**
1. Go to your profile
2. Got to create skill.
3. See error

**LogCat for the issue**

**Screenshots of the issue**
Screenshot 2020-01-31 02:22:38

**Would you like to work on the issue?**
Yes , working on it

### PR DESCRIPTION
**Pull request by mohitm15**
Fixes #3403 

**Changes:**
I changed the styling of the text written on skill card and bot card .

**Screenshots of the change:**

**Before**
Screenshot 2020-01-31 02:22:38

Screenshot 2020-02-01 18:54:57

**After**

Screenshot 2020-02-14 02:47:15

Screenshot 2020-02-14 02:44:32

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@material-ui/lab": "3.0.0-alpha.30",
     "add-px": "1.0.0",
     "axios": "0.19.0",
+    "babel-eslint": "^10.0.3",
```
Comment (snitin315) [inline package.json]: Theese changes are not related to the issue. Please make changes according to issue only .

File: src/components/cms/SkillCreator/SkillCreator.js [hunk 2]
```diff
@@ -297,7 +297,16 @@ class SkillCreator extends Component {
...
                     />
                   </Fab>
-                  <CardContent>Create a new skill</CardContent>
+                  <CardContent
+                    style={{
+                      fontSize: '1.5rem',
+                      color: 'black',
+                      textShadow: '2px 2px 8px white',
+                      fontWeight: 'bold',
+                    }}
```
Comment (snitin315) [inline src/components/cms/SkillCreator/SkillCreator.js]: Avoid custom css. It's not a good practice.

Comment (mohitm15) [reply src/components/cms/SkillCreator/SkillCreator.js]: It's a mere change only, that's why added inline styling.

File: package.json (hunk 1 above)
Comment (mohitm15) [reply package.json]: Actually this is coming always when I commit the below changes. I can only commit after passes by babel-linter.

File: src/components/cms/SkillCreator/SkillCreator.js (hunk 2 above)
Comment (snitin315) [reply src/components/cms/SkillCreator/SkillCreator.js]: We are using Material UI which already comes with lot of pre-defined css. We should make use of it.

File: package.json (hunk 1 above)
Comment (snitin315) [reply package.json]: Fix it.  `+14814` lines of change is not appericiated.

Comment (mohitm15) [reply package.json]: Ok, I remove `yarn.lock`

**Comment by tobehonest** [comment]
@mohitm15  Please add a surge link too. Its hard to tell how the button looks with other page elements with just an Image.

File: src/components/cms/SkillCreator/SkillCreator.js (hunk 2 above)
Comment (mohitm15) [reply src/components/cms/SkillCreator/SkillCreator.js]: @snitin315 So Should I use Material UI or leave same for this small change

Comment (snitin315) [reply src/components/cms/SkillCreator/SkillCreator.js]: Check if there is a predefined class/component for same , if found use it.

Comment (mohitm15) [reply src/components/cms/SkillCreator/SkillCreator.js]: ok

**Comment by mohitm15** [comment]
@snitin315 @akshatnitd Now review this

File: package.json [hunk 3]
```diff
@@ -87,6 +88,7 @@
     "redux-actions": "2.6.5",
     "redux-mock-store": "1.5.3",
     "redux-promise": "0.6.0",
+    "standard": "^14.3.1",
```
Comment (snitin315) [inline package.json]: @mohitm15  Don't add this file to staging area.

**Comment by mohitm15** [comment]
I didn't add them for staging. But before pushing the commit was getting
failed .So after fetching , these changes comes automatically.

**Comment by mohitm15** [comment]
Should I open another PR for this and close the previous one

**Comment by mohitm15** [comment]
> Should I open another PR for this and close the previous one
>
What should I do? @Orbiter @akshatnitd

**Comment by snitin315** [comment]
> Should I open another PR for this and close the previous one
 
No this is a very bad practice. If you are having trouble in reading the branch just reset your branch with upstream and force push the changes again.

```
git pull --rebase upstream master
git reset --hard upstream/master
```
Then make the required changes again. Commit changes.

`git push --force origin master `

**Comment by mohitm15** [comment]
@snitin315 I know that , earlier I forgot to add yarn.lock file in .gitignore , that's why faced issue.

**Comment by snitin315** [comment]
Run these commands again , This should act as a New PR for you I guess

**Comment by mohitm15** [comment]
Ok

**Comment by mohitm15** [comment]
> Run these commands again , This should act as a New PR for you I guess

Do I need to change the branch ?

**Comment by snitin315** [comment]
No

**Comment by mohitm15** [comment]
> No

but it is again asking me to pull first before pushing it

**Comment by snitin315** [comment]
Share screenshot

**Comment by mohitm15** [comment]
Sure

**Comment by mohitm15** [comment]
> Share screenshot

Screenshot 2020-02-23 19:01:25

**Comment by snitin315** [comment]
use `--force`

**Comment by mohitm15** [comment]
@akshatnitd Please review this

**Comment by akshatnitd** [comment]
Closing due to inactivity.
